RNPComponents/RNPComponents.md

```diff
@@ -0,0 +1,476 @@
+
+    <Button
+        mode="elevated"
+        buttonColor={theme.colors.surfaceContainerLow}
+    >
+        Label
+    </Button>
+    <Button
+        mode="elevated"
+        textColor={theme.colors.error}
+        buttonColor={theme.colors.surfaceContainerLow}
+    >
+        Label
+    </Button>
+    <Button
+        mode="elevated"
+        disabled
+        buttonColor={theme.colors.surfaceContainerLow}
+    >
+        Label
+    </Button>
+    ```
+
+  - #### Contained Tonal Mode
+
+    Recommended style overrides:
+    * text color of the error text button: onErrorContainer
+    * button color of the error text button: errorContainer
+
+    <img width='300' alt="Button Contained-tonal Mode" src="./Images/buttonTonal.png"><br/>
+
+    Usage
+    ```tsx
+    import { Button } from 'react-native-paper';
+
+    <Button mode="contained-tonal">
+        Label
+    </Button>
+    <Button
+        mode="contained-tonal"
+        buttonColor={theme.colors.errorContainer}
+        textColor={theme.colors.onErrorContainer}
+    >
+        Label
+    </Button>
+    <Button mode="contained-tonal" disabled>
+        Label
+    </Button>
+    ```
+
+- ###  Data Table
+
+    Recommended style overrides:
+    * Text color of Table Title: onSurface
+    * font size of Table Title: theme.fonts.labelLarge
+
+    <img width='300' alt="Data Table" src="./Images/dataTable.png"><br/>
+
+    Usage
+    ```tsx
+    import { DataTable } from 'react-native-paper';
+
+    <DataTable>
+        <DataTable.Header>
+            <DataTable.Title
+                textStyle={{ color: theme.colors.onSurface, ...theme.fonts.labelLarge }}
+            >
+                Dessert
+            </DataTable.Title>
+            <DataTable.Title
+                numeric
+                textStyle={{ color: theme.colors.onSurface, ...theme.fonts.labelLarge }}
+            >
+                Calories
+            </DataTable.Title>
+            <DataTable.Title
+                numeric
+                textStyle={{ color: theme.colors.onSurface, ...theme.fonts.labelLarge }}
+            >
+                Fat
+            </DataTable.Title>
+        </DataTable.Header>
+
+        {items.slice(from, to).map((item) => (
+            <DataTable.Row key={item.key}>
+                <DataTable.Cell>{item.name}</DataTable.Cell>
+                <DataTable.Cell numeric>{item.calories}</DataTable.Cell>
+                <DataTable.Cell numeric>{item.fat}</DataTable.Cell>
+            </DataTable.Row>
+        ))}
+
+        <DataTable.Pagination
+            page={page}
+            numberOfPages={Math.ceil(items.length / itemsPerPage)}
+            label={`${from + 1}-${to} of ${items.length}`}
+            numberOfItemsPerPageList={numberOfItemsPerPageList}
+            numberOfItemsPerPage={itemsPerPage}
+            onItemsPerPageChange={onItemsPerPageChange}
+            showFastPaginationControls
+            selectPageDropdownLabel={'Rows per page'}
+        />
+    </DataTable>
```

Comment:
Sorry I just noticed this. It appears that the table cell and the pagination is not getting the appropriate typography -- note the shape of `C`, `S` and `R`

![image](https://github.com/etn-ccis/blui-react-native-themes/assets/8997218/161cd759-788d-4f08-92f5-c35a9822fe37)

Reply:
https://github.com/etn-ccis/blui-react-native-showcase-demo/issues/181